pkg/asset/cluster/aws/aws.go

```diff
@@ -69,11 +69,15 @@ func tagSharedVPCResources(ctx context.Context, clusterID string, installConfig
 	}
 
 	tagKey, tagValue := sharedTag(clusterID)
+	elbTagKey, elbTagValue := elbTag()
 
 	ec2Client := ec2.New(session, aws.NewConfig().WithRegion(installConfig.Config.Platform.AWS.Region))
 	if _, err = ec2Client.CreateTagsWithContext(ctx, &ec2.CreateTagsInput{
```

Comment:
For the call to `ChangeTagsForResourceWithContext` we should be consistent with the contexts and make a child context before passing it to the API.